charts/govuk-rails-app/values.yaml

```diff
@@ -11,21 +11,17 @@ externalSecrets:
 
 ingress:
   enabled: false
-  name:
-  host: ""
-  path: "/"
-  pathType: Prefix
   annotations:
     alb.ingress.kubernetes.io/scheme: internet-facing
     alb.ingress.kubernetes.io/target-type: ip
     alb.ingress.kubernetes.io/listen-ports: '[{"HTTP": 80}, {"HTTPS": 443}]'
     alb.ingress.kubernetes.io/ssl-redirect: "443"
     alb.ingress.kubernetes.io/healthcheck-path: /healthcheck/ready
+    alb.ingress.kubernetes.io/load-balancer-name:
```

Comment:
Probably better to leave load-balancer-name out than set it to empty string, since it'd likely cause problems if anyone were to actually use the example as-is. If you really wanted to keep it for illustrative purposes then you could leave it commented out with an example name set.

(Also a small nit in case it comes up elsewhere: if you need to set one of these values to empty string, it's better to write `foo: ""` rather than just `foo: ` so that it's strictly valid YAML. Helm seems to put up with it but it's a good idea to stick to the spec. Not sure why lint isn't catching it 🤷)
```suggestion
```